Auth0Tests/AuthenticationSpec.swift

```diff
@@ -728,27 +759,27 @@ class AuthenticationSpec: QuickSpec {
                     }
                 }
             }
-
+            
```

Comment:
Only change recently in editor was upgrading to 8.3, generally with tests I run  `Structure/Re-Indent`